plugin/trino-plugin-archetype/src/main/resources/archetype-resources/src/main/java/__classPrefix__ColumnHandle.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package $package;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.trino.spi.connector.ColumnHandle;
+import io.trino.spi.type.Type;
+
+import java.util.Objects;
+
+public class ${classPrefix}ColumnHandle
```

Comment:
This is going to be a pain to maintain. You can't edit, run and debug using a normal java editor and any refactorings in the main Trino codebase won't propagate to these files.

We used to have an archetype builder for an airlift-based server in airlift that worked by taking the sample/skeleton server and replace packages and other relevant names as a post-processing step. See https://github.com/airlift/airlift/blob/f992559581a13a33f8a68e9b3977f429718ec039/skeleton-server-archetype-builder/pom.xml. That might be a better model if we want to do this.